CMakeLists.txt

```diff
@@ -31,8 +31,7 @@ option(GIT_SUBMODULE "Check submodules during build" ON)
 option(DEBUG_INFO "Print message for developers to debug." OFF)
 # Do not enable it if generated code will run on different CPUs
 option(ENABLE_NATIVE_OPTIMIZATION "Enable compilation optimization for the native machine's CPU type" OFF)
-# Do not use the new container by default.
-option(ENABLE_CONTAINER "Enable the new multi-device container." OFF)
+option(ENABLE_CONTAINER "Enable the new multi-device container." ON)
```

Comment:
Do you want to keep this option, which means both new container tensor and old case should be both keeped in code.

Reply:
I suggest retaining this option as the container is currently in the development stage. We will gradually utilize the container to replace the multi-device ops in the original code.

Reply:
Deleted with the latest commit.